src/components/SiteNav/index.astro

```diff
@@ -1,30 +1,67 @@
 ---
 import { Image } from 'astro:assets';
 import logo from '../../assets/icons/logo.svg';
+import closeIcon from './assets/close-icon.svg';
+import hamburgerIcon from './assets/hamburger-menu.svg';
 import './styles.css';
 const BLOG_URL = Astro.site?.href;
 ---
-<nav id="site-nav">
-	<ul>
+
+<nav id="site-nav" aria-label="Main navigation menu">
+	<ul id="nav-menubar">
 		<li>
 			<a href={BLOG_URL}>
 				<Image src={logo} alt="TorontoJS Logo!" height={70} width={70} id="logo-rounded" />
 			</a>
 		</li>
-		<li>
-			<a href={`${BLOG_URL}about`}>
-				About
-			</a>
-		</li>
-		<li>
+		<li class="desktop-only">
 			<a href={`${BLOG_URL}accessibility`}>
 				Accessibility
 			</a>
 		</li>
-		<li>
+		<li class="desktop-only">
 			<a href={`${BLOG_URL}license`}>
 				License
 			</a>
 		</li>
+		<li class="desktop-only">
+			<a href={`${BLOG_URL}about`}>
+				About
+			</a>
+		</li>
+		<li class="mobile-only">
+			<button
+				id="nav-menu-button" aria-controls="mobile-menu" aria-expanded="false" aria-haspopup="true"
+				aria-label="Open blog navigation" popovertarget="mobile-menu"
+			>
+				<Image
+					id="hamburger-icon" src={hamburgerIcon} alt="Hamburger Menu Icon" width={70} height={70}
+				/>
```

Comment:
Please use the `<Icon />` component instead, it will inline and SVG image and can give better control and accessibility.

Here is an example of it's syntax: https://github.com/torontojs/blog/blob/main/src/components/Pagination/index.astro#L38

Docs are here: https://www.astroicon.dev/guides/customization/

We already have a couple of icon sets that are installed (they are on `package.json`), so check them out first if the icons you are looking for are available. you can check the full icon sets here: https://icon-sets.iconify.design/